providers/ArtistFeedProvider.tsx

```
@@ -1,19 +1,38 @@
+  const filtered = canEdit
+    ? feeds.feeds
+    : feeds.feeds.filter(
+        (feed) =>
+          !Boolean(
+            hiddenMoments.find(
+              (moment) =>
+                moment.tokenContract.toLowerCase() ===
+                  feed.collection.toLowerCase() &&
+                moment.tokenId === feed.tokenId,
+            ),
+          ),
```

Biome (1.9.4)

[error] Avoid redundant `Boolean` call

It is not necessary to use `Boolean` call when a value will already be coerced to a boolean.
Unsafe fix: Remove redundant `Boolean` call


(lint/complexity/noExtraBooleanCast)